### ISSUE TITLE
doesn't work with Waterline Sqlite3 adapter

### ISSUE THREAD

**Issue by ScottMaxson** [seabourne/nxus-dataset#11]
Various errors are thrown when loading data into, and manipulating the included `datasets-datarow` model using the waterline-sqlite3 adapter. The database representation of this model is created without a reference to the parent `datasets-dataset`; queries on the datarow's `dataset` column are built by the adapter but fail. An example:

     [Error (E_UNKNOWN) Encountered an unexpected error] Details:  Error: DELETE FROM "datasets-datarow" WHERE "datasets-datarow"."dataset" = '1'  - SQLITE_ERROR: no such column: datasets-datarow.dataset

This may be related to the design of the `datasets-dataset` model which doesn't include the parent side of the association.

**Comment by loppear** [comment]
I wouldn't expect it to need the parent ref to be defined in this foreignKey case. I'd be curious a) what is the schema of this table in sqlite3 b) is this an in-memory sqlite db, or is it possible that waterline earlier created this schema (with a diff name?) and won't update it?

**Comment by ScottMaxson** [comment]
Yes, me either ... Good questions: I'll try dumping the schema. My latest attempt to reproduce this issue started with a fresh Sqlite db (I removed the `.tmp/default.sqllite` file before starting up the app), and I saw the issue right away.

**Comment by ScottMaxson** [comment]
sqlite3 client prints the following as the CREATE statement used for the `datasets-datarow` table:
```
    CREATE TABLE "datasets-datarow" ("id" integer, "createdAt" integer, "updatedAt" integer, primary key ("id"));
```
